Simplenote/NoteListViewController.swift

```diff
@@ -591,7 +591,7 @@ extension NoteListViewController {
 
     @IBAction
     func performSearch(_ sender: Any) {
-        searchQuery = SearchQuery(searchText: searchField.stringValue)
+        searchQuery = SearchQuery(searchText: searchField.stringValue, settings: .default)
```

Comment:
@jleandroperez @eshurakov folks, I'm not sure if this is correct. If it isn't hopefully you're able to send a GitHub suggestion?

Thanks 🙇

Reply:
No need to change anything over here [because of this](https://github.com/Automattic/simplenote-macos/blob/develop/Simplenote/SearchQuery+Simplenote.swift#L6)

There's a convenience initializer for SearchQuery, which does this automagically. (Sent you over a code suggestion!).

**Thank you Gio!!**